qsiprep/reports/core.py

```diff
@@ -24,7 +24,7 @@
 
 from nireports.assembler.report import Report
 
-from .. import data
+from qsiprep import data
```

Comment:
does ruff prefer named imports?

Reply:
Actually this fails if you enable doctests. I was originally going to enable them in this PR but there were a bunch of ones failing so I disabled that.